config/manifests/bases/horizon-operator.clusterserviceversion.yaml

```diff
@@ -6,8 +6,11 @@ metadata:
     capabilities: Basic Install
     features.operators.openshift.io/disconnected: "true"
     features.operators.openshift.io/fips-compliant: "true"
-    operatorframework.io/suggested-namespace: openstack
-    operators.openshift.io/infrastructure-features: '["disconnected"]'
+    features.operators.openshift.io/token-auth-gcp: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    feeatures.operators.openshift.io/token-auth-aws: "false"
```

Comment:
`s/feeatures/features`

Reply:
It's almost as if I just copy and pasted them :p